packages/plugins/typescript/operations/tests/ts-documents.spec.ts

```diff
@@ -2847,16 +2848,16 @@ export type Q2Query = { search: Array<
       });
 
       expect(content).toBeSimilarStringTo(
-        `export type TestQueryQueryVariables = Exact<{
-          username?: InputMaybe<Scalars['String']['input']>;
-          email?: InputMaybe<Scalars['String']['input']>;
-          password: Scalars['String']['input'];
-          input?: InputMaybe<InputType>;
+        dedent(`export type TestQueryQueryVariables = Exact<{
+          username?: string | null;
+          email?: string | null;
+          password: string;
+          input?: InputType | null;
           mandatoryInput: InputType;
-          testArray?: InputMaybe<Array<InputMaybe<Scalars['String']['input']>> | InputMaybe<Scalars['String']['input']>>;
-          requireString: Array<InputMaybe<Scalars['String']['input']>> | InputMaybe<Scalars['String']['input']>;
-          innerRequired: Array<Scalars['String']['input']> | Scalars['String']['input'];
-        }>;`
+          testArray?: Array<string | null> | string | null;
```

Comment:
Huh interesting, TIL we can pass an `String` into an array of `String` in the variables